### PR TITLE
Adding better colors to differentiate

### PR DESCRIPTION
**Pull request by rowlandm**
I need to be able to tell between students who I have graded as being interesting versus those who are slightly less so. This is based on the 01 to 08 scale of students and their skills.

Would be great to review this @ZIYUT and @Xenotion and @ labib08

### CODE REVIEW

File: static/styles.css [hunk 1]
```diff
@@ -21,13 +21,19 @@ Kanban boards styles
     color: white ! important;
 }
 
+.partially-accepted {
+    background-color: #70A3CC !important;
```
Comment (Xenotion) [inline static/styles.css:25]: Inconsistent code styling

should there be a gap between '!' and 'important' like line 21

**Review by Xenotion — APPROVED** [review]
Looks good to me!

File: static/styles.css (hunk 1 above)
Comment (rowlandm) [reply static/styles.css:25]: Updated - thanks for letting me know :) 

I'm sure there are other improvements to the code that could be done.

PS. Why hasn't anyone replied to their WEHI emails this week? I think we need to ensure that we can work independently from you (as you can't work in the day) but also together, if you know what I mean....

**Comment by rowlandm** [comment]
Close

**Pull request merged**